region/wreckedship/main.json

```diff
@@ -457,24 +457,24 @@
                 {
                   "name": "Shoot the Blocks",
                   "notable": false,
-                  "requires": [
-                    {"or": [
-                      "canLateralMidAirMorph",
-                      "h_canUseSpringBall"
-                    ]}
-                  ],
+                  "requires": [ "Morph" ],
+                  "note": "Simple jump morph from the top of the stairs.",
                   "obstacles": [
                     {
                       "id": "B",
                       "requires": [
-                        {"or":[
-                          "Wave",
-                          "Spazer"
-                        ]}
-                      ]
-                    }
-                  ],
-                  "note": "It's possible to lateral mid-air morph into the morph tunnel."
+                        {"or": [
+                            "Wave",
+                            "Spazer",
+                            {"and": [
+                                "canUseFrozenEnemies",
+                                "canCeilingClip",
+                                {"not": [ "f_DefeatedPhantoon" ]}
+                              ]}
+                          ]}
+                      ],
+                      "note": "Use the Covern to partial ceiling clip so your beam can reach the obstacle shot block."
```

Comment:
This note will be shown the the player, even if they have wave or spazer. Maybe start the note with something like "Use Wave or Spazer to shoot the blocks, otherwise use the Covern ... "?

Reply:
Yeah it could be worth having a separate strat for using the Covern